app/lib/proxy_api/ansible.rb

```diff
@@ -53,5 +53,61 @@ def playbooks(playbooks_names = [])
     rescue *PROXY_ERRORS => e
       raise ProxyException.new(url, e, N_('Unable to get playbooks from Ansible'))
     end
+
+    def repo_information(vcs_url)
+      parse(get("vcs_clone/repo_information?vcs_url=#{vcs_url}"))
+    rescue *PROXY_ERRORS, RestClient::Exception => e
+      raise e unless e.is_a? RestClient::RequestFailed
```

Comment:
I'd only catch the specific exception:
```suggestion
    rescue RestClient::RequestFailed => e
```

Reply:
I had something different in my mind, but just now realized that your suggestion works fine and is cleaner...
I'll add it after the new changes have been reviewed.